docs/usage_guide.rst

```diff
@@ -817,7 +817,10 @@ Adding and Removing Blockchain Accounts
 
 rotki allows to track balances of blockchain accounts.
 
-To add or modify an account navigate to the "Blockchain Balances" sub-page and click the large "+" icon. Now choose the blockchain on which you want to add an account (for now only Bitcoin, Bitcoin Cash, Ethereum, Kusama, Polkadot and Avalanche chains are supported). Then type or paste the address in the "Account" textbox and press the "Save" Button. Note that you can add multiple accounts if you click the "Add multiple addresses" checkbox and provide a comma separated list of addresses.
+To add or modify an account navigate to the "Blockchain Balances" sub-page and click the large "+" icon. Now choose the blockchain on which you want to add an account (for now only Bitcoin, Bitcoin Cash, Ethereum, Optimism, Kusama, Polkadot and Avalanche chains are supported). Then type or paste the address in the "Account" textbox and press the "Save" Button. Note that you can add multiple accounts if you click the "Add multiple addresses" checkbox and provide a comma separated list of addresses.
+
+If the selected chain is an EVM chain you will see "Add to all supported EVM chains" checkbox. It is checked by default and it means that rotki will try to add the address for all EVM chains. If the address is a contract in Ethereum mainnet it will only be added to Ethereum. Otherwise for each chain rotki will check whether the address had any activity there and will add only if it has at least one transaction (or in case of Avalanche if the addres has a non-zero balance). If you uncheck the checkbox, then the address will only be added to the selected chain.
```

Comment:
Though that is true, it is a hopefully temporary implementation detail so I would just not mention it.

```suggestion
If the selected chain is an EVM chain you will see "Add to all supported EVM chains" checkbox. It is checked by default and it means that rotki will try to add the address for all EVM chains. If the address is a contract in Ethereum mainnet it will only be added to Ethereum. Otherwise for each chain rotki will check whether the address had any activity there and will add only if it has at least one transaction. If you uncheck the checkbox, then the address will only be added to the selected chain.
```

Reply:
The one thing I asked is still not resolved
![2023-01-26_17-23](https://user-images.githubusercontent.com/1658405/214890853-8e61a947-c78d-4064-b028-f9088c186312.png)